fineract-provider/src/main/java/org/apache/fineract/portfolio/client/domain/Client.java

```diff
@@ -763,30 +750,18 @@ public boolean isStaff() {
         return this.isStaff;
     }
 
-    public String getExternalId() {
-        return this.externalId;
+    public void setExternalId(ExternalId externalId) {
```

Comment:
There should be no setter for external id. Why do we need this?

Reply:
Removed